api/taskcluster/webhook.go

```diff
@@ -293,7 +328,6 @@ func getAuth(ctx context.Context) (username string, password string, err error)
 func createAllRuns(
 	log shared.Logger,
 	aeAPI shared.AppEngineAPI,
-	checksAPI checks.API,
```

Comment:
This was a circular dep, and was only used for pending checks (which we have disabled for now).
Pending checks will be re-implemented (if at all) with lifecycle - see #1151